src/components/node/Node.jsx

```diff
@@ -98,9 +100,18 @@ export default class Node extends React.Component {
         let label;
         let node;
 
+        //Determine if we have a rectangle or a square size
+        var sizeWidth = size;
```

Comment:
It's a good practice in JS to avoid `var` whenever possible and use `let` or `const` instead. In this case, `let` is the way to go. :wink:

Reply:
fixed